app/models/concerns/offer_search.rb

```diff
@@ -12,7 +12,6 @@ module OfferSearch
     # package_type: name_en, name_zh_tw
 
     scope :search, -> (options = {}) {
-      offer_states = options[:states].empty? ? Offer.nondraft_states : options[:states]
```

Comment:
doesn't that remove the ability to filter by state altogether ? `options[:states]` seems to be ignored completely.
Maybe I'm no understanding the PR right 🤔

Reply:
I have added state filter code in offer filtering (https://github.com/crossroads/api.goodcity/blob/master/app/models/concerns/offer_filtering.rb#L14) so there was duplication of state filters. In offer filtering it will be required in future for dashboard states. So, I thought it would be better to remove from it from `search_filtering.rb`